frontend/src/pages/coding/[id].tsx

```diff
@@ -199,16 +247,26 @@ const CodingPage = () => {
 						/>
 						<h1 className="font-bold text-xl">{project?.title}</h1>
 					</div>
-					<div className="flex items-center self-end md:self-center">
-						<Button
-							size={"sm"}
-							className="gap-2 bg-blue-500 hover:bg-blue-500/80"
-							onClick={saveCode}
-						>
-							<Save />
-							Enregistrer
-						</Button>
-					</div>
+					{project && userId === project.user.id && (
+						<div className="flex items-center self-end md:self-center gap-5">
+							<div className="flex items-center space-x-2">
+								<Switch
+									id="public-state"
+									checked={project.isPublic}
+									onCheckedChange={handlePublicStateChange}
+								/>
+								<Label htmlFor="public-state">En Public</Label>
+							</div>
+							<Button
+								size={"sm"}
+								className="gap-2 bg-blue-500 hover:bg-blue-500/80"
+								onClick={saveCode}
+							>
+								<Save />
+								Enregistrer
+							</Button>
```

Comment:
```suggestion
							<Button
								size={"sm"}
								className="gap-2"
								onClick={saveCode}
								variant={"dark"}
							>
								<Save />
								Enregistrer
							</Button>
```
Quand le variant dark sera disponible, je trouve que ça colle mieux aux couleurs du site que le bleu
![image](https://github.com/user-attachments/assets/269c1426-4bba-46d3-81f1-e20fb26256fd)

Reply:
Okok ben je testerai ça avec le variant dark, mais oui sur le screen c'est pas mal